src/lib/accessibility/Behaviors/List/ListItemBehavior.ts

```diff
@@ -1,13 +1,13 @@
 import SelectableListItemBehavior from './SelectableListItemBehavior'
 import BasicListItemBehavior from './BasicListItemBehavior'
-import { IAccessibilityDefinition } from '../../interfaces'
+import { Accessibility } from '../../interfaces'
 
 /**
  * @description
  * Defines a behavior "BasicListItemBehavior" or "SelectableListItemBehavior" based on "selection" property.
  */
 
-const ListItemBehavior: (props: any) => IAccessibilityDefinition = (props: any) =>
-  props.selection ? SelectableListItemBehavior(props) : BasicListItemBehavior
+const ListItemBehavior: Accessibility = (props: any) =>
+  props.selection ? SelectableListItemBehavior(props) : BasicListItemBehavior(props)
```

Comment:
To me it feels kind a awkward that all accessibility behaviors are defined as functions but they are starting with capital letter.

Reply:
From point of view that functions should be lowercase - yes agree with that. 
But for me, it seems better as it is now, and I can't explain why :) 
If anyone also thinks about changing to lowercase, of course, we can consider it.